hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -1,61 +1,64 @@
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 joint_limits:
-  right_wrist_2_joint:
+  left_elbow_joint:
     has_velocity_limits: true
-    max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  right_wrist_3_joint:
+    max_velocity: 3.15
+    has_acceleration_limits: false
+    max_acceleration: 0
+  left_shoulder_lift_joint:
     has_velocity_limits: true
-    max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  left_elbow_joint:
+    max_velocity: 3.15
+    has_acceleration_limits: false
+    max_acceleration: 0
+  left_shoulder_pan_joint:
     has_velocity_limits: true
     max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
+    has_acceleration_limits: false
+    max_acceleration: 0
   left_wrist_1_joint:
     has_velocity_limits: true
     max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  left_wrist_3_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  left_wrist_2_joint:
     has_velocity_limits: true
     max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  right_shoulder_lift_joint:
-    has_velocity_limits: true
-    max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
-  left_wrist_2_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  left_wrist_3_joint:
     has_velocity_limits: true
     max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  left_shoulder_pan_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_elbow_joint:
     has_velocity_limits: true
     max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
-  left_shoulder_lift_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_shoulder_lift_joint:
     has_velocity_limits: true
     max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
-  right_elbow_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_shoulder_pan_joint:
     has_velocity_limits: true
     max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
+    has_acceleration_limits: false
+    max_acceleration: 0
   right_wrist_1_joint:
     has_velocity_limits: true
     max_velocity: 3.2
-    has_acceleration_limits: true
-    max_acceleration: 5
-  right_shoulder_pan_joint:
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_wrist_2_joint:
     has_velocity_limits: true
-    max_velocity: 3.15
-    has_acceleration_limits: true
-    max_acceleration: 5
\ No newline at end of file
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_wrist_3_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
Ah I missed that over-write from the Setup Assistant, hadn't realized they were different. I can add those back